src/theme/global-style.ts

```diff
@@ -8,7 +8,6 @@ export const globalStyle: SystemStyleObject = {
     flexDirection: 'column',
     minHeight: '100%',
     color: tokens.gray800,
-    overflowX: 'hidden',
```

Comment:
Would you mind moving this to the `<main />` element and see if that breaks anything - there's a reason we added this but I can't find what it was. I'd like to keep it at least on the main content, and perhaps the `header` as well?

Reply:
I can move it to main, then the banner would be below the nav bar. I think you were fixing the cookie text being overflow-x. That part is fixed now. In the other templates, we don't have overflow-x:hidden set either. We can double double it during QA to ensure this is fine.

Reply:
I meant moving the `overflow-x` to the main element. But if you're sure it's fixed - then we can ignore that